apps/condo/domains/acquiring/tasks/recurrent-payment-seeking-for-new-receipt-task.js

```diff
@@ -0,0 +1,7 @@
+const { createCronTask } = require('@open-condo/keystone/tasks')
+
+const { process } = require('@condo/domains/acquiring/tasks/recurrent-payment-seeking-for-new-receipt/recurrent-payment-seeking-for-new-receipt')
+
+module.exports = createCronTask('recurrentPaymentSeekingForNewReceipt', '0 * * * *', async () => {
```

Comment:
you can move it inside index

Reply:
Moved to implementation file (as I can see for other tasks)